apps/ecommerce/lambda/src/config.ts

```diff
@@ -26,4 +26,5 @@ export const config = {
   sentryDSN: getEnvironmentVariable('SENTRY_DSN'),
   environment: getEnvironmentVariable('NODE_ENV'),
   release: getEnvironmentVariable('CIRCLE_SHA1', 'no-release-hash-set'),
+  baseUrl: getEnvironmentVariable('BASE_URL'),
```

Comment:
For now this work but it brings up that we eventually need to retrieve this data from the provider installation itself. That is, if you installed Magento app it should find a different provider base URL than Shopify, i.e. to point to those different provider instances.

Reply:
yep, absolutely